launcher/container-runner.service

```diff
@@ -6,7 +6,8 @@ After=network-online.target gcr-online.target containerd.service
 [Service]
 ExecStart=/var/lib/google/cc_container_launcher --addr=${ATTEST_ENDPOINT}
 # Shutdown the host after the launcher exits
-ExecStopPost=systemctl poweroff
+ExecStopPost=/bin/sleep 60
```

Comment:
I tried looking through the documentation (https://www.freedesktop.org/software/systemd/man/systemd.service.html) but couldn't find anything about ordering. Does systemd guarantee execution in order or declaration?